pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -551,9 +533,55 @@ func (cm *containerManagerImpl) Start(node *v1.Node, activePods ActivePodsFunc)
 		}, 5*time.Minute, wait.NeverStop)
 	}
 
+	// Local storage filesystem information from `RootFsInfo` and `ImagesFsInfo` is availalbe at a later time
+	// depending on the time when cadvisor manager updates container stats. Therefore use a go routine to keep
+	// retrieving the information until it is avaialble within 5 minutes timeout.
+	timeout := 5 * time.Minute
+	go cm.setFsCapacity(timeout)
+
 	return nil
 }
 
+func (cm *containerManagerImpl) setFsCapacity(timeout time.Duration) {
+	timeoutTimer := time.NewTimer(timeout)
+	interval := 1 * time.Second
+	defer timeoutTimer.Stop()
+	for {
+		select {
+		case <-timeoutTimer.C:
```

Comment:
You can use wait.Forever if you dont want a timeout, or wait.Until with a chan that closes after a timeout if you wanted one.